app/external_services/github_repos_api.rb

```diff
@@ -0,0 +1,15 @@
+class GithubReposApi
+  URL = 'https://api.github.com/repos/rootstrap'.freeze
+
+  def initialize(project_name)
+    @project_name = project_name
+  end
+
+  def get_content_from_file(file_name)
+    response = Faraday.get("#{URL}/#{@project_name}/contents/#{file_name}", {},
+                           'Accept' => 'application/vnd.github.v3.raw')
+    return response.body if response.success?
+
+    {}
```

Comment:
In this case would invert the early return to

```ruby
return {} unless response.success?
response.body
```

returning early for anything but the happy path mentioned in the method name.

Or since this conditional is a straight forward accessor with not logic maybe

```ruby
response.success? ? response.body : {}
```